insights/specs/default.py

```diff
@@ -228,7 +228,7 @@ def dnf_module_names(broker):
             return [m for m in dml]
         raise SkipComponent()
 
-    dnf_module_info = foreach_execute(dnf_module_names, "/usr/bin/dnf -C --noplugins module info %s", deps=IsRhel8)
+    dnf_module_info = foreach_execute(dnf_module_names, "/usr/bin/dnf -C --noplugins module info %s", deps=[IsRhel8])
```

Comment:
@xiangce thanks for working on this parser. Do I understand it correctly (as the function name suggests) that for every module name we execute "dnf module info $module"?

I just tried to run "dnf module info module1 module2 module3" with list of modules and it works just fine and should be faster. What do you think?

Reply:
@Rezney - Yes, it works. Thanks for the suggestion!

Reply:
Hey @Rezney, the dependency PR #2112 was merged. Then I have updated the `dnf module info` parser as per your suggestion. Please have a look at the update.

Reply:
@xiangce looks good to me from "dnf" perspective...thanks